src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/ReadResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,56 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Read" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify existing resources can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify retrieving a single resource by ID
+        # Replace Endpoint with the required value, this is just an example to make sure Descriptors are working fine
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "id": "{id}"
+                        "codeValue": "Jury duty",
+                        "description": "Jury duty",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Jury duty"
+                        "_etag": "5250182869515346067",
+                        "_lastModifiedDate": "2024-04-25T23:38:28.7958163Z"
+                    }
+                  """
+
+        @ignore
+        Scenario: Verify response code 404 when ID does not exist
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/123123123123"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: Verify array records content on GET All
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+              And total of records should be greather than "0"
```

Comment:
Let's be precise here: there is only one record in the database when we reach this point, so we can say that the total should be exactly 1.

Reply:
Fixing the previous comment I was going to ask about the total or records, I was thinking that there were some pre-existing records in the DB, but that being said I will update this validation. Thanks

Reply:
Interesting... you have a good point here. I was planning on loading the sample database with an initial set of descriptors, so that you don't have to specify all of the descriptors in a Background or Given section. If this were a regular Resource, like `students`, instead of a Descriptor, then the total would be exactly 1. But since it is a Descriptor, there will be existing records.

That also means that the example of "jury duty" should be made unique - it shouldn't be a real value that we would normally have in the API. Otherwise your initial attempt to POST the data will find that the record already exists.

Reply:
@stephenfuqua  Do we need to update/edit something here?

Reply:
Once we finish these tests, we might need to change the codeValue to something truly unique. Not necessary right now.